shards/modules/physics/physics.hpp

```diff
@@ -0,0 +1,217 @@
+#ifndef ACE82164_D020_40B6_A9E1_A13621726A27
+#define ACE82164_D020_40B6_A9E1_A13621726A27
+
+#include <shards/core/object_var_util.hpp>
+#include <shards/core/assert.hpp>
+#include <shards/core/exposed_type_utils.hpp>
+#include <shards/core/hasherxxh3.hpp>
+#include <shards/core/async.hpp>
+#include <shards/common_types.hpp>
+#include <shards/log/log.hpp>
+#include <Jolt/Jolt.h>
+#include <Jolt/Core/Reference.h>
+#include <Jolt/Physics/Collision/Shape/Shape.h>
+#include <Jolt/Physics/Body/Body.h>
+#include <Jolt/Physics/SoftBody/SoftBodySharedSettings.h>
+#include <Jolt/Math/MathTypes.h>
+#include <Jolt/Math/Quat.h>
+#include <boost/container/flat_map.hpp>
+#include <boost/container/stable_vector.hpp>
+#include <tracy/Wrapper.hpp>
+#include <linalg.h>
+#include <memory>
+#include <atomic>
+
+namespace shards {
+inline SHVar toVar(const JPH::Float3 &f3) {
```

Comment:
For functions that return values, like toVar, toJPHFloat3, etc., consider adding the [[nodiscard]] attribute to ensure that the return value is not accidentally ignored.

Reply:
Might also add `noexcept`

Reply:
Oke